src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts

```diff
@@ -261,6 +263,23 @@ export class InterfacesFormComponent implements OnDestroy {
     this.vlan_pint = _.find(this.fieldConfig, {'name' : 'vlan_parent_interface'});
     this.bridge_members = _.find(this.fieldConfig, {'name' : 'bridge_members'});
     this.lag_ports = _.find(this.fieldConfig, {'name' : 'lag_ports'});
+
+    if (window.localStorage.getItem('is_freenas') === 'false') {
+      this.ws.call('failover.node').subscribe((node) => {
+        if (node === 'A') {
+          this.ipPlaceholder = ' (This Controller)';
```

Comment:
Shouldn't this use the global text here?

Reply:
I updated 'This Controller'  to use global variable.